lfr/7pin_cardinal_demo/runjob_sawtooth.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#PBS -N HCFwork
+#PBS -l select=4:ncpus=36:mpiprocs=36:mem=128GB:ompthreads=1
```

Comment:
sawtooth has nodes thta are bigger than that. this isnt optimal

Reply:
Correct, I am not sure why not all 48 cores are used. Possible memory restriction? Anyways, but this is just a example script for potential users, and they may or may not have to use Sawtooth.